.test-infra/jenkins/job_PreCommit_Java.groovy

```diff
@@ -41,7 +41,7 @@ builder.build {
         checkStyle {
           pattern('**/build/reports/checkstyle/*.xml')
         }
-        findBugs {
+        spotBugs {
```

Comment:
https://github.com/spotbugs/spotbugs-gradle-plugin/blob/master/src/main/java/com/github/spotbugs/SpotBugsPlugin.java#L122

Should be `spotbugs`.

It seems that extension names are case insensitive, but it may be still worth changing.

Reply:
Thanks for catching this one!